docker-compose.yml

```diff
@@ -25,40 +24,40 @@ services:
       - alice-lightningd:/data/.lightning
       - lsp-lightningd:/data/.lightning-lsp
 
-  bitcoind:
+  bitcoind-breez:
     command:
-      -addresstype=bech32
-      -debug=mempool
-      -debug=rpc
-      -fallbackfee=0.00000253
-      -logtimestamps
-      -nolisten
-      -regtest
-      -rpcallowip=0.0.0.0/0
-      -rpcbind=0.0.0.0
-      -rpcpassword=btcpass
-      -rpcuser=btcuser
-      -rest
-      -server
-      -txindex
+      - -addresstype=bech32
+      - -debug=mempool
+      - -debug=rpc
+      - -fallbackfee=0.00000253
+      - -logtimestamps
+      - -nolisten
+      - -regtest
+      - -rpcallowip=0.0.0.0/0
+      - -rpcbind=0.0.0.0
+      - -rpcpassword=btcpass
+      - -rpcuser=btcuser
+      - -rest
+      - -server
+      - -txindex
+      - -addnode=boltz-bitcoind:18444
```

Comment:
I wonder whether that works on regtest environments? Especially because both bitcoin nodes will be mining blocks.

Reply:
I didn't see any issues yet. If, during tests, only one of the nodes is mining, it should be fine.